code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -462,14 +465,54 @@
 	if(user.a_intent == INTENT_HARM)
 		return ..()
 	if(locked)
-		to_chat(user, "<span class='warning'>The maintenance panel is locked.</span>")
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
 		return TRUE // Must be true or we attempt to stab the bot
 
 	open = !open
 	I.play_tool_sound(src)
 	to_chat(user, "<span class='notice'>The maintenance panel is now [open ? "opened" : "closed"].</span>")
 	return TRUE
 
+/mob/living/simple_animal/bot/wrench_act(mob/living/user, obj/item/I)
+	if(user.a_intent != INTENT_HELP)
+		return..()
+	if(locked) //ID-Lock
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if(!open) //Screwdriver panel
+		to_chat(user, "<span class='warning'>The maintenance panel is screwed in place.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if((bot_type != SEC_BOT) && (bot_type != GRIEF_BOT)) //Should only affect security bots and griefsky
+		to_chat(user, "<span class='warning'>The internal components cannot be wrenched more securely.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+
+	I.play_tool_sound(src)
+	wrenched = !wrenched
+	to_chat(user, "<span class='notice'>The internal components are now [wrenched ? "wrenched" : "unwrenched"].</span>")
+	return TRUE
```

Comment:
```suggestion
	. = TRUE // Must be true or we attempt to bash the bot
	if(user.a_intent != INTENT_HELP)
		return ..()
	if(locked) //ID-Lock
		to_chat(user, "<span class='warning'>The interface is locked.</span>")
		return
	if(!open) //Screwdriver panel
		to_chat(user, "<span class='warning'>The maintenance panel is screwed in place.</span>")
		return
	if((bot_type != SEC_BOT) && (bot_type != GRIEF_BOT)) //Should only affect security bots and griefsky
		to_chat(user, "<span class='warning'>The internal components cannot be wrenched more securely.</span>")
		return

	I.play_tool_sound(src)
	wrenched = !wrenched
	to_chat(user, "<span class='notice'>The internal components are now [wrenched ? "wrenched" : "unwrenched"].</span>")
```
You can set the default return value to be `TRUE` if you want everything to return `TRUE`!

Reply:
I am not fully understanding your code. Please do correct me if I am wrong :

`. = TRUE` sets the default return value of the multitool's code block as `TRUE`.
And `return ..()` would call the predefined proc value for return, so the one you defined with `. = TRUE`. But if that is the case then why are the subsequents `return` not using the `..()` ?

Reply:
Oooh. Okay I finally understand your code. It is indeed better than writing `return = TRUE` everywhere.

Reply:
(I somehow had not made the link between setting the default return value and writing the sugsequents returns without any overrides..)